packages/x-components/src/views/home/aside.vue

```diff
@@ -6,7 +6,7 @@
       <template #default="{ filter }">
         <SimpleFilter :filter="filter" class="x-facet-filter-success">
           <template #label>
-            {{ filter.label ?? `${filter.range.min} - ${filter.range.max}` }}
+            {{ filter.label ?? filter.id }}
```

Comment:
The id is set as an alternative label. If the min/max range were used, when the page is reloaded or we perform another search, the filter's getter won't keep that values as the filter model changes to a RawFilter.